docs/paper/admin/reference/world-configuration.md

```diff
@@ -166,7 +166,7 @@ this list.
 ### fix-climbing-bypassing-cramming-rule
 
 - **default**: `false`
-- **description**: Sets whether climbing should bypass the entity cramming limit.
+- **description**: Sets whether climbing should bypass the entity cramming limit(`maxEntityCramming` game rule). If set to `true`, climbing entities will also be counted toward the entity cramming limit so that they can take suffocation damage.
```

Comment:
toward or towards? s sounds better in my head but not a native

Reply:
According to [this article](https://www.grammarly.com/blog/toward-towards/), both `toward` and `towards` are acceptable. But `toward` is preferred in the US and Canada, `towards` is preferred in the UK and Australia.  
I'm not a native either, but I personally prefer `toward`.

Reply:
The 'rest of the docs' (only one instance tbf) use towards, might be better to use it for consistency's sake

Reply:
fuck @electronicboy got me used to british 
(thanks for checking)

Reply:
Well, if then, I will change it to `towards`. :)